frontend/src/Admin/components/ShowSubmissionsTab.js

```diff
@@ -45,7 +52,18 @@ class ShowSubmissionsTab extends Component {
 
     this.state = {
       alertVisible: true,
-      alertType: ''
+      alertType: '',
+      showFinalized: false,
```

Comment:
I don't think you need this?

Shouldn't you be able to get the updated value from `props`?